okhttp/src/main/kotlin/okhttp3/internal/connection/RealConnection.kt

```diff
@@ -26,7 +26,7 @@ import java.util.concurrent.TimeUnit.MILLISECONDS
 import java.util.concurrent.locks.ReentrantLock
 import javax.net.ssl.SSLPeerUnverifiedException
 import javax.net.ssl.SSLSocket
-import kotlin.concurrent.withLock
```

Comment:
no longer need when the references from `lock.withLock` were changed to `this.withLock`
 - though that extension function still forwards to `lock.withLock` so same concurrent method used one step further.